src/Button.tsx

```diff
@@ -463,14 +463,15 @@ export interface ButtonProps
   [key: string]: any; // bad hack to permit component injection
 }
 
+interface Button extends React.ForwardRefExoticComponent<ButtonProps & React.RefAttributes<HTMLButtonElement>> {
```

Comment:
This interface is explicitly defined by extending `React.ForwardRefExoticComponent` because `React.ForwardRefExoticComponent` does not have `propsTypes`.
Ref: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/34757#issuecomment-488848720